CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.8)
 
-project(Thrust CXX)
+project(Thrust NONE)
```

Comment:
This is going to be broken for non-CUDA backends, no? Because the only `enable_language(CXX)` I can see below is inside the block for the CUDA backend.

Reply:
Yah good point, fixing.